test/functional/outputs/cloudwatch/forward_to_cloudwatch_test.go

```diff
@@ -148,13 +180,13 @@ var _ = Describe("[Functional][Outputs][CloudWatch] Forward Output to CloudWatch
 			if testfw.LogCollectionType == logging.LogCollectionTypeVector {
```

Comment:
We should investigate removing since there is no need to have special handling for vector.  Either the test applies as-is (which is what I would expect) or the test can be removed entirely